PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetProperties.h

```diff
@@ -0,0 +1,233 @@
+
+  TH1F *fh1DelRRCJet1;
+  TH1F *fh1DelRRCJet2;
+  TH1F *fh1nRC;
+
+  //BkgSubtracted
+  TProfile* hProDelRPtDenUEN;
+  TProfile* hhProDelRPtSumUE[21];       //!Pt sum vs R
+  TProfile* hhProDelRPtDenUE[21];       //!Pt density vs R
+  TH1F*     fh1rho;                   //!rho distribution
+  TH1F*     fh1rhoN;                   //!rho for Ntracks distribution
+  TH1F*     h1LeadingJetPtWUE;        //!highest jet pt
+  TH1F*     h1LeadingJetPtWoUEWoCAll;       //!highest jet pt after UE subtraction
+  TProfile* fProNtracksLeadingJetWUE; //!number of tracks in leading jet
+  TProfile* fProNtracksLeadingJetUEWoCAll; //!number of tracks in leading jet (UE)
+  TProfile* fProNtracksLeadingJetWoUEWoCAll;//!number of tracks in leading jet after UE subtraction
+  TProfile* fProNtracksLeadingJetUEwrtSubtPtWoC; //!number of tracks in leading jet (UE) wrt subt jet pt
+  TProfile* fProNtracksLeadingJetWoUEwrtSubtPtWoC;//!number of tracks in leading jet after UE subtraction wrt subt jet pt
+
+  TH1F      *h1Cfactor; // C factor
+  TH1F*     h1LeadingJetPtWoUEWCAll;       //!highest jet pt after UE subtraction
+  TProfile* fProNtracksLeadingJetUEWCAll; //!number of tracks in leading jet (UE)
+  TProfile* fProNtracksLeadingJetWoUEWCAll;//!number of tracks in leading jet after UE subtraction
+  TProfile* fProNtracksLeadingJetUEwrtSubtPtWC; //!number of tracks in leading jet (UE) wrt subt jet pt
+  TProfile* fProNtracksLeadingJetWoUEwrtSubtPtWC;//!number of tracks in leading jet after UE subtraction wrt subt jet pt
+
+  THnSparseD* h4ResNch;
+  TH2D* h2NchGen;
+  TH2D* h2NchRec;
+
+  THnSparseD* h4ResNchUE;
+  TH2D* h2NchGenUE;
+  TH2D* h2NchRecUE;
+
+  THnSparseD* h4ResFF;
+  TH2D* h2FFGen;
+  TH2D* h2FFRec;
+  THnSparseD* h4ResFFM;
+  TH2D* h2FFMFake;
+  TH2D* h2FFMMiss;
+  TH2D* h2FFMGen;
+  TH2D* h2FFMRec;
+
+  THnSparseD* h4ResFFMUE;
+  TH2D* h2FFMUEFake;
+  TH2D* h2FFMUEMiss;
+  TH2D* h2FFMUEGen;
+  TH2D* h2FFMUERec;
+
+  THnSparseD* h5ResPtSum;
+  THnSparseD* h5ResPtSumUE;
+
+  TH2D* h2ResJetPt;
+  TH2D* h2ResJetPtNch;
+
+   //______________________HM_________________________________//                                                                                              
+  Double_t  fMultV0A;                                  //!  mult. V0A                                                                                        
+  Double_t  fMultV0C;                                  //!  mult. V0C                                                                                        
+  Double_t  fMultV0M;                                  //!  mult. V0A+V0C                                                                                    
+
+  TH1D* fhistMultV0A;
+  TH1D* fhistMultV0C;
+  TH1D* fhistMultV0M;
+
+  //______________________HM_________________________________//                                                                                               
+
+
+  /// \cond CLASSIMP
+  ClassDef(AliAnalysisTaskEmcalJetProperties, 1);
```

Comment:
Since the streamer comments are incomplete (`//!<!` for unstreamed members vs `///<` for streamed members), I would worry whether everything works as expected. I guess you've run this submitting jobs directly to the grid, so I suppose it does work, but best practices would be to fill in the comments for all of your members

Reply:
Yes, I used this code in the grid for job submission. Could you please explain what you are suggesting?

Reply:
Okay, lets leave this out.